src/components/ebay-toggle-button/toggle-button.stories.js

```diff
@@ -23,66 +25,71 @@ export default {
     },
     argTypes: {
         renderBody: {},
-        gallery: {
-            type: "boolean",
-            control: { type: "boolean" },
-            description: "true when the button is in gallery layout",
+        layoutType: {
+            type: "string",
+            control: { type: "select" },
+            options: ["minimal", "list", "gallery"],
+            description:
+                'Enforced layout type of the button. May be `"minimal"` (default), `"list"`, or `"gallery"`',
         },
         pressed: {
             type: "boolean",
             control: { type: "boolean" },
-            description: "pressed state of the button",
+            description: "Pressed state of the button",
         },
         title: {
             type: "string",
             control: { type: "text" },
-            description: "title attribute for the button",
+            description: "Title attribute for the button",
         },
         subtitle: {
-            type: "string",
+            type: "string|@subtitle",
             control: { type: "text" },
-            description: "subtitle attribute for the button",
+            description: "Subtitle attribute for the button",
         },
         icon: {
             name: "@icon",
-            description: "an `<ebay-[name]-icon>` to show as the button's icon",
+            description: "An `<ebay-[name]-icon>` to show as the button's icon",
             table: {
                 category: "@attribute tags",
             },
         },
         img: {
             name: "@img",
             description: "An `<img>` to show as the button's image",
+            table: {
+                category: "@attribute tags",
+            },
         },
-        href: {
+        subtitleTag: {
+            name: "@subtitle",
+            description:
+                "May be used instead of the `subtitle` attribute for more control. Should contain no more than two brief lines of text",
             table: {
-                category: "@img attributes",
+                category: "@attribute tags",
             },
-            control: { type: "text" },
-            description: "href attribute for the image",
         },
-        alt: {
+        src: {
             table: {
                 category: "@img attributes",
             },
             control: { type: "text" },
-            description: "alt attribute for the image",
+            description: "Link to the image source",
         },
-        size: {
+        alt: {
             table: {
                 category: "@img attributes",
             },
             control: { type: "text" },
-            description:
-                "size of the image. Values may be any which are applicable to the CSS `background-size` property, but only `contain` and `cover` are fully supported.",
+            description: "Alt text for the image",
         },
-        position: {
+        fillPlacement: {
```

Comment:
In the Description for the option, we should include the fact that this is the `css` `background-position` and that it can take positioning words, percentages, and pixels. 
<img width="1180" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/cc94efe7-d4c0-42b4-97d6-80c2d886ea1e">